### PR TITLE
Prepare for v0.8.0 release

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 <!-- markdownlint-disable no-duplicate-heading -->
 # Change Log
 
+## 0.8.0
+
+### Minor Changes
+
+* Added support for meta/runtime.yml and execution-environment.yml (#379) @ssbarnea
```
Comment (webknjaz) [inline CHANGELOG.md:8]: Let's highlight these
```suggestion
* Added support for `meta/runtime.yml` and `execution-environment.yml` (#379) @ssbarnea
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+* Added schema for ansible-navigator configuration (#365) @ssbarnea
```
Comment (webknjaz) [inline CHANGELOG.md:9]:
```suggestion
* Added a schema for ansible-navigator configuration (#365) @ssbarnea
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+
+### Bugfixes
+
+* Declared lextudio.restructuredtext ext as conflicting (#366) @ssbarnea
```
Comment (webknjaz) [inline CHANGELOG.md:13]: Writing the changelog for humans means that we should avoid shortening/colloquial speech. This is also emphasized in the style guide that the Ansible tech writers use.
```suggestion
* Declared `lextudio.restructuredtext` extension as conflicting (#366) @ssbarnea
```

**Comment by webknjaz** [comment]
@ssbarnea is it possible to tell Jenkins to release from this branch and only then merge the PR here?

**Comment by webknjaz** [comment]
> delaying this release for next week as is already late friday.

Fair enough. #no_friday_releases

**Comment by ssbarnea** [comment]
> > delaying this release for next week as is already late friday.
> 
> Fair enough. #no_friday_releases

I hope we can also fix https://github.com/ansible/vscode-ansible/issues/373 which is the bug that hindered @priyamsahoo e2e pull-request. We worked around it but the root cause is still very valid.

**Review by ganeshrn — COMMENTED** [review]
Can we also include the recent commit in this release?

**Comment by ssbarnea** [comment]
> Can we also include the recent commit in this release?

Can you please be more explicit? I do not see anything else listed on https://github.com/ansible/vscode-ansible/releases other than the already included 3 items.

Also, I am inclined to delay this release until we fix #373 which counts as a serious bug.

**Pull request closed without merging**